File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.c
int countConsistentStrings(char * allowed, char ** words, int wordsSize){

int flag=0;
int sum=0;

for(int i=0;i<wordsSize;i++)
{
    int length = strlen(words[i]);
    for(int j=0;j<length;j++)
    {
        if(strchr(allowed,words[i][j])==NULL)
        {
            flag=1;
        }
    }
    
    if(flag==0)
    {
        sum++;
    }
    flag=0;
}

return sum;
}

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        int total = 0;
        int sum = 0;
        for (int i = 0; i < words.length;i++){
            boolean control = true;
            for (int j = 0; j < words[i].length();j++){
                if (allowed.indexOf(words[i].charAt(j)) < 0){
                    control = false;
                }
            }
            if (control == true){
                sum += 1;
            }
        }
        return sum;  
    }
}